tools/retest/main.go

```diff
@@ -2,102 +2,280 @@ package main
 
 import (
 	"context"
+	"fmt"
 	"log"
 	"net/http"
 	"os"
+	"regexp"
+	"slices"
+	"strconv"
 	"strings"
+	"time"
 
 	"github.com/google/go-github/v60/github"
 )
 
-func main() {
-	ctx := context.Background()
-
-	retestComment := "/retest"
+const s = "stackrox"
 
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
+	defer cancel()
 	// Use installation transport with client.
```

Comment:
I just realized I don't understand what this comment means..

Reply:
```suggestion
```